src/Containers/Sidebar/Sidebar.js

```diff
@@ -1,26 +1,65 @@
 import React, { Component } from 'react';
 import ProjectArea from '../ProjectArea/ProjectArea';
+import { postProject } from '../../Thunks/postProject';
+import { connect } from 'react-redux';
 
 export class Sidebar extends Component{
-  constructor (){
+  constructor() {
     super();
     this.state = {
-      name: ''
+      name: '',
+      error: ''
+    };
+  };
+  
+  handleChange = (event) => {
+    const { value } = event.target;
+    this.setState({ name: value });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { name } = this.state;
+    const { projects } = this.props;
+    const foundName = projects.find(project => {
+      return name.toLowerCase() === project.name.toLowerCase()
+    })
+    if (foundName) {
```

Comment:
Good check here to make sure the project name doesn't match another one, since the db won't allow matching project names